core/src/tvu.rs

```diff
@@ -191,7 +192,11 @@ impl Tvu {
         );
 
         let (verified_sender, verified_receiver) = unbounded();
-        let (retransmit_sender, retransmit_receiver) = unbounded();
+
+        // Allow for a max of 2048 batches of up to 1024 packets each (according to MAX_IOV).
+        // In reality this holds about 20K shreds since most batches are not full.
```

Comment:
nit: I would prefer to define this as a constant.

Also, thoughts on making this way larger (e.g. 32k) just to make sure we only drop in extreme cases? Hopefully 20k shreds is a single block before too long :)

Reply:
This will allow the buffer to store about 3 GB of data. Not sure how great of an idea it is, but I guess we can play it safe for now.